e2e/volume_test.go

```diff
@@ -95,7 +95,7 @@ func TestPodWithMultiVolume(t *testing.T) {
 	}
 	t.Log("success create pod with multi-volume")
 
-	cmd = kubectl("exec", "pod/vk-e2e-volume", "--namespace=vk-test", "--", "ls /var/run/secrets/kubernetes.io/serviceaccount")
+	cmd = kubectl("exec", "pod/vk-e2e-volume", "-c", "hpa-example", "--namespace=vk-test", "--", "ls /var/run/secrets/kubernetes.io/serviceaccount")
```

Comment:
Did we change the container name?

Reply:
No, there are two containers and it does default to this container but I thought it might be better to specify it. I can remove it if you think its not necessary